docs/security/csrfProtection.md

```diff
@@ -0,0 +1,15 @@
+Our approach to prevent or at least maximum decrease chances for any CSRF attacks
```

Comment:
Documentation looks good but this file can be removed if we go with JSDoc next to the method itself

Reply:
done